### ISSUE TITLE
More hyperparameters management for training

### ISSUE THREAD

**Issue by fabian57fabian** [ultralytics/hub#384]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and found no similar feature requests.

### Description

When training a model from Ultralytics HUB, there are only a few hyperparameters available (e.g. epochs, patience ...) and only one "advanced" hp for batch size.
Could we add more? Like all Augmentation params, or starting learning rate...
Alternatively an input text box where to put a json/yaml file with all overriding params, similar to yaml config file.

### Use case

During model training interface. Could be used to set more hyperparameters.

**Comment by UltralyticsAssistant** [comment]
@fabian57fabian thank you for your suggestion on enhancing the hyperparameters control during the model training phase!

Your idea to include more hyperparameters like augmentation parameters, learning rate, or providing an input text box to include a YAML/JSON file with overriding parameters certainly adds more flexibility to the customization process. It's an interesting approach to make the platform more adaptable to different user needs.

We highly appreciate the feedback from our users and the YOLO community and we constantly strive to improve our platform based on that. We will put forward your suggestion to our development team for evaluation. Keep an eye out for updates on our GitHub page for the implementation of this feature.

Thank you again for your insightful suggestion!

**Comment by fabian57fabian** [comment]
I'll keep an eye but if no changes will be made and other people don't require this, it could be closed.

**Comment by UltralyticsAssistant** [comment]
@fabian57fabian, thank you for your understanding. We appreciate the community's patience and cooperation as we decide upon improvements to implement on the Ultralytics HUB. As always, user feedback is crucial to our development, and we truly value your suggestions.

Even if it takes a while for your specific request to be addressed, we hope that you and others will continue to see the many benefits of our platform. Rest assured that we consider every suggestion and work to implement those ideas that best fit our roadmap and can benefit the most users.

Again, we appreciate your input and understanding. Please don't hesitate to contribute in the future with any other feature requests or issues you may have.
